docs/src/npzd_model_benchmark.md

```diff
@@ -0,0 +1,208 @@
+# [Benchmark: Solution of an NPZD model](@id benchmark-npzd)
+
+We use the NPZD model [`prob_pds_npzd`](@ref) to assess the efficiency of different solvers.
+
+Standard methods have difficulties to solve this problem accurately, at least for low tolerances.
+
+```@example NPZD
+using OrdinaryDiffEq, PositiveIntegrators
+using Plots
+
+# select problem
+prob = prob_pds_npzd
+
+# compute reference solution (standard tolerances are too low)
+ref_sol = solve(prob, Vern7(); abstol = 1e-14, reltol = 1e-13)
+
+# compute solutions with low tolerances
```

Comment:
```suggestion
# compute solutions with lose tolerances
```

We have had confusion about this. For us, the best option so far has been to talk about strict tolerances like `1e-12` and lose tolerances like `1e-3`.